src/shared/kilocode/cli-sessions/core/SessionManager.ts

```diff
@@ -374,6 +419,10 @@ export class SessionManager {
 			}
 		}
 
+		if (process.env.KILO_DISABLE_SESSIONS) {
```

Comment:
@RSO @catrielmuller @pandemicsyn I noticed integration tests are creating sessions - once this is merged, they should probably set this env var.